### ISSUE TITLE
[Request] Add crossing wires symbol

### ISSUE THREAD

**Issue by mm2595** [circuitdiagram/components#75]
Please add "connected" and "not connected" symbols.

**Comment by Corsaka** [comment]
...not connected = no connection visible.
![image](https://user-images.githubusercontent.com/23699128/75049225-bf1d2100-54c1-11ea-8294-1a6ba847da68.png)


connected = connection visible.
![image](https://user-images.githubusercontent.com/23699128/75049204-b75d7c80-54c1-11ea-9835-c71f1552cd2c.png)

**Comment by mm2595** [comment]
Hi,


Thanks for your reply. I am not quite sure how to make connections
invisible. Every time I make two wires crossed, there is a dot. Can you
help me with that?






Corsaka <notifications@github.com>于2020年2月21日 周五上午10:49写道：

**Comment by Corsaka** [comment]
Connections will always be shown by a dot. If the wires are not connected, there will not be a dot. To overlay two wires without the dot showing, make sure that the end of one wire is not on the other one.

**Comment by dyllbert** [comment]
While I agree it does implement this feature, it would be nice for clarity to be able to manually insert the rounded "not connected" symbol something like this:

![contact-nocontact](https://user-images.githubusercontent.com/36219518/75464845-06cdfd80-5945-11ea-87e0-60bc1289fe9e.png)

**Comment by SamuelFisher** [comment]
I don't think this fits as a "component" however I will leave this issue open to keep track of this feature.
